### PR TITLE
chore(useDataValue): Add internal `updateValue`

### PR DESCRIPTION
**Pull request by tujoworker**

### CODE REVIEW

**Review by langz — APPROVED** [review]

**Review by henit — CHANGES_REQUESTED** [review]

File: packages/dnb-eufemia/src/extensions/forms/hooks/useDataValue.ts [hunk 1]
```diff
@@ -398,31 +399,52 @@ export default function useDataValue<
       // Always validate the value immediately when it is changed
       validateValue()
 
-      onChange?.(newValue)
```
[the remark this answers is not in the document]
Comment (tujoworker) [reply packages/dnb-eufemia/src/extensions/forms/hooks/useDataValue.ts:401]: The only motivation with updateValue is to "not" call onChange during a state change from outside. But it does work technically. onChange should be tied to a user event I would argument.

[the remark this answers is not in the document]
Comment (tujoworker) [reply packages/dnb-eufemia/src/extensions/forms/hooks/useDataValue.ts:401]: `changedRef` is used for a focus event handling. I'm not sure if this should be set to true i this case. What do you think?

**Comment by tujoworker** [comment]
Had a call with @henit and this are looking good 👍

**Pull request merged**